src/org/jitsi/impl/neomedia/transform/rewriting/SsrcGroupRewriter.java

```diff
@@ -66,7 +68,8 @@
      * A map of SSRCs to <tt>SsrcRewriter</tt>. Each SSRC that we rewrite in
      * this group rewriter has its own rewriter.
      */
-    private final Map<Integer, SsrcRewriter> rewriters = new HashMap<>();
+    private final Map<Integer, SsrcRewriter> rewriters
+        = new ConcurrentHashMap<>();
```

Comment:
Same thing here, the `rewriters` object is accessed by a single thread at the time. We should be clear about that tho, to avoid confusion in the future.